### PR TITLE
fix ResourceControlQueue::updateStatistics declaration

### PR DESCRIPTION
**Pull request by guo-shaoge**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by windtalker — APPROVED** [review]
LGTM

**Review by SeaRise — APPROVED** [review]

File: dbms/src/Common/ThreadedWorker.h [hunk 1]
```diff
@@ -204,8 +204,8 @@ class ThreadedWorker
             auto error = getCurrentExceptionMessage(false);
             LOG_ERROR(log, "{}#{} meet error: {}", getName(), thread_idx, error);
             auto cancel_reason = fmt::format("{} failed: {}", getName(), error);
-            result_queue->cancelWith(cancel_reason);
             source_queue->cancelWith(cancel_reason);
+            result_queue->cancelWith(cancel_reason);
```
Comment (guo-shaoge) [inline dbms/src/Common/ThreadedWorker.h:208]: 42d6a878-4885-40bb-a647-2d2c20fd8a9c
Change order of cancel to fix above unittest. Check [code](https://github.com/pingcap/tiflash/blob/master/dbms/src/Common/tests/gtest_threaded_worker.cpp#L309)

Comment (guo-shaoge) [reply dbms/src/Common/ThreadedWorker.h:208]: /cc @breezewish

**Pull request merged**